libsrc/osic1p/bootstrap.s

```diff
@@ -45,6 +45,8 @@ GETCHAR         :=      $FFBF           ; gets one character from ACIA
 FIRSTVISC       =       $85             ; Offset of first visible character in video RAM
 LINEDIST        =       $20             ; Offset in video RAM between two lines
 
+        lda     #0
+        sta     load
```

Comment:
You are correct.

The code increments the load address partly in an index register and partly in RAM.  But, that wasn't my original code.  My first version incremented the `count` variable in both a register and RAM.  Then, I realized that I could get more efficiency by doing it with `load` instead of `count`.  So, I editted the source until I had what seemed to work.  But, starting from the old source, instead of an empty screen, caused me to make two mistakes:
* I forgot that the _base_ value of `load` needs a valid low byte too.
* I left this inefficient piece of code in the program:
```GAS
  lda #<load_addr
  tay
```
(I should have loaded .Y directly!)

There are two ways to initiate the load incrementer:
 1. Put zero into the low byte of `load`.  And, put `<load_addr` into the Y register.
 2. Put zero in the Y register.  And, put `<load_addr` into `load`.

The second way is traditional.  I would have done it that way if my mind had not been "corrupted" by my original version.  So, this is how I would write it now:
```GAS
ldy  #<$0000
lda  #<load_addr
ldx  #>load_addr
sta  load
stx  load+1
```
(That zero is written in a style that Oliver and I like.  It shows that the register is being used to hold the lower byte of a 16-bit value.)